server/src/main/java/io/openbytes/odi/domain/repository/DatasetRepository.java

```diff
@@ -16,6 +16,8 @@
 
 package io.openbytes.odi.domain.repository;
 
+import com.baomidou.mybatisplus.core.metadata.IPage;
+import com.baomidou.mybatisplus.extension.plugins.pagination.Page;
```

Comment:
Should there be no dependency on implementation classes in the interface layer？

Reply:
yes and done.

Reply:
👿